catalog/resourcemanagement/resourcemanagement-usage-ui/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -0,0 +1,40 @@
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="
+       http://www.osgi.org/xmlns/blueprint/v1.0.0 http://www.osgi.org/xmlns/blueprint/v1.0.0/blueprint.xsd
+       http://camel.apache.org/schema/blueprint http://camel.apache.org/schema/blueprint/camel-blueprint.xsd">
+
+    <bean id="spatialPlugin" class="org.codice.ddf.resourcemanagement.usage.DataUsagePlugin" />
+
+    <bean id="dataUsage" class="org.codice.ddf.resourcemanagement.usage.service.DataUsage" init-method="init">
+        <property name="attributesStore" ref="attributesStoreService" />
+        <property name="persistentStore" ref="persistentStore" />
+    </bean>
+
+    <reference id="attributesStoreService" interface="org.codice.ddf.persistence.attributes.AttributesStore" />
+
+    <reference id="persistentStore" interface="org.codice.ddf.persistence.PersistentStore"/>
```

Comment:
Might want to add `Service` to the end of this id since it's a bit confusing that one of your `reference`s is named that way but the other is not.